xdmod-appkernels.spec.in

```diff
@@ -64,22 +64,22 @@ rm -rf $RPM_BUILD_ROOT
 
 %changelog
 * Tue Apr 23 2019 XDMoD <ccr-xdmod-list@listserv.buffalo.edu> 8.1.0-1.0
-- Updated for compatibility with Open XDMoD 8.1.0.
+- Release 8.1.0.
 * Wed Nov 07 2018 XDMoD <ccr-xdmod-list@listserv.buffalo.edu> 8.0.0-1.0
-- Updated for compatibility with Open XDMoD 8.0.0.
+- Release 8.0.0.
 * Thu Mar 01 2018 XDMoD <ccr-xdmod-list@listserv.buffalo.edu>7.5.0-1.0
-- Updated for compatibility with Open XDMoD 7.5.0.
+- Release 7.5.0.
 * Thu Sep 21 2017 XDMoD <ccr-xdmod-list@listserv.buffalo.edu>7.0.0-1.0
-- Updated for compatibility with Open XDMoD 7.0.0.
+- Release 7.0.0.
 * Thu May 11 2017 XDMoD <ccr-xdmod-list@listserv.buffalo.edu>6.6.0-1.0
-- Updated for compatibility with Open XDMoD 6.6.0.
+- Release 6.6.0.
 * Tue Jan 10 2017 XDMoD <ccr-xdmod-list@listserv.buffalo.edu>6.5.0-1.0
-- Updated for compatibility with Open XDMoD 6.5.0.
+- Release 6.5.0.
 * Wed Sep 21 2016 XDMoD <ccr-xdmod-list@listserv.buffalo.edu>6.0.0-1.0
-- Updated for compatibility with Open XDMoD 6.0.0.
+- Release 6.0.0.
 * Tue May 24 2016 XDMoD <ccr-xdmod-list@listserv.buffalo.edu>5.6.0-1.0
-- Updated for compatibility with Open XDMoD 5.6.0.
+- Release 5.6.0.
 * Fri Dec 18 2015 XDMoD <ccr-xdmod-list@listserv.buffalo.edu>5.5.0-1.0
-- Updated for compatibility with Open XDMoD 5.5.0.
+- Release 5.5.0.
```

Comment:
Compare with https://github.com/ubccr/xdmod/blob/xdmod8.1/open_xdmod/modules/xdmod/xdmod.spec.in#L125